merkato/exchanges/tux_exchange/exchange.py

```diff
@@ -5,54 +5,92 @@
 import requests
 import time
 import urllib.parse
-
+from merkato.exchanges.tux_exchange.utils import getQueryParameters
 from merkato.exchanges.exchange_base import ExchangeBase
-
+from merkato.constants import BUY, SELL
 DEBUG = True
 
 
 class TuxExchange(ExchangeBase):
     url = "https://tuxexchange.com/api"
 
-    def __init__(self, auth):
+    def __init__(self, auth, config):
         self.privatekey = auth['privatekey']
         self.publickey  = auth['publickey']
-
-    def sell(self, amount, ask, ticker):
+        self.limit_only = config.limit_only
+        self.retries = 5
```

Comment:
Are we still doing the retries thing? Theres the order placed after read but during retries issue to deal with still (can still merge this and fix later)